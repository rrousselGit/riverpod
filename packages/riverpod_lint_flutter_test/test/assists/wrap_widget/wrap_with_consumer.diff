Message: `Wrap with Consumer`
Priority: 28
Diff for file `test/assists/wrap_widget/wrap_widget.dart:11`:
```
    Map();

-     return Scaffold(
-       body: Container(),
-     );
+     return Consumer(builder: (context, ref, child) { return Scaffold(
+       body: Container(),
+     ); },);
  }
}
```
---
Message: `Wrap with Consumer`
Priority: 28
Diff for file `test/assists/wrap_widget/wrap_widget.dart:12`:
```

    return Scaffold(
-       body: Container(),
+       body: Consumer(builder: (context, ref, child) { return Container(); },),
    );
  }
```
---
