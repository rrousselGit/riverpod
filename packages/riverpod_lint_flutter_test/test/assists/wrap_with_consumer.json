[{"priority":28,"change":{"message":"Wrap with Consumer","edits":[{"fileStamp":0,"edits":[{"offset":281,"length":0,"replacement":"; },)"},{"offset":241,"length":0,"replacement":"Consumer(builder: (context, ref, child) { return "}]}],"linkedEditGroups":[]}},{"priority":28,"change":{"message":"Wrap with Consumer","edits":[{"fileStamp":0,"edits":[{"offset":274,"length":0,"replacement":"; },)"},{"offset":263,"length":0,"replacement":"Consumer(builder: (context, ref, child) { return "}]}],"linkedEditGroups":[]}}]