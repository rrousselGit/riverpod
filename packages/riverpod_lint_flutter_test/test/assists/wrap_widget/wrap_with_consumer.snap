Message: `Wrap with Consumer`
<<<< start: 241 -- end: 279
Scaffold(
      body: Container(),
   
==== start: 241 -- end: 333
Consumer(builder: (context, ref, child) { return Scaffold(
      body: Container(),
    ); }
>>>>
Message: `Wrap with Consumer`
<<<< start: 266 -- end: 272
tainer
==== start: 266 -- end: 326
sumer(builder: (context, ref, child) { return Container(); }
>>>>
